sbin/wifibox

```diff
@@ -703,9 +709,39 @@ vm_manager() {
 
     log debug "Nice priority: ${_nice_priority}"
     log debug "Arguments: ${_bhyve_args}"
-    ${NICE} -n ${_nice_priority} \
-	    ${BHYVE} ${_bhyve_args} 2>&1 | capture_output debug bhyve
+
+    _kill_child() {
```

Comment:
Please do not use tabs for indentation in the `_kill_child` function.  All of them have to translated to spaces, that is how the rest of the file is written.

Reply:
My understanding is that for each function, the indentation begins with 4 spaces, then a second indentation is a tab, a third a tab + 4 spaces, a fourth two tabs, etc. 

Nonetheless, I've used all spaces now, and fixed some other incorrect indentation in the file.